### ISSUE TITLE
Images with alt text shows up twice

### ISSUE THREAD

**Issue by mgeisler** [google/mdbook-i18n-helpers#31]
See https://google.github.io/comprehensive-rust/pt-BR/, which currently looks like this:

![image](https://github.com/google/mdbook-i18n-helpers/assets/89623/4026d437-f2a2-40f5-a933-5ebdfad68ba9)

The underlying HTML looks weird, with the images being duplicated:

```html
<p>
<a href="https://github.com/google/comprehensive-rust/actions/workflows/build.yml?query=branch%3Amain">
  <img src="https://img.shields.io/github/actions/workflow/status/google/comprehensive-rust/build.yml?style=flat-square" alt="Build workflow">
</a>
Build workflow
<a href="https://github.com/google/comprehensive-rust/actions/workflows/build.yml?query=branch%3Amain">
  <img src="https://img.shields.io/github/actions/workflow/status/google/comprehensive-rust/build.yml?style=flat-square" alt="Build workflow">
</a>
<a href="https://github.com/google/comprehensive-rust/graphs/contributors">
  <img src="https://img.shields.io/github/contributors/google/comprehensive-rust?style=flat-square" alt="GitHub contributors">
</a>
GitHub contributors
<a href="https://github.com/google/comprehensive-rust/graphs/contributors">
  <img src="https://img.shields.io/github/contributors/google/comprehensive-rust?style=flat-square" alt="GitHub contributors">
</a>
<a href="https://github.com/google/comprehensive-rust/stargazers">
  <img src="https://img.shields.io/github/stars/google/comprehensive-rust?style=flat-square" alt="GitHub stars">
</a>
GitHub stars
<a href="https://github.com/google/comprehensive-rust/stargazers">
  <img src="https://img.shields.io/github/stars/google/comprehensive-rust?style=flat-square" alt="GitHub stars">
</a>
</p>
```

**Comment by mgeisler** [comment]
I tested this kind of Markdown with the current `main` branch and it's fixed there. The problem above seems to be the nested tags: the image inside the link causes us to extract things twice.

**Comment by mgeisler** [comment]
This was fixed with the 0.2.0 release. Please upgrade and run `mdbook-i18n-normalize` on your PO files.
